src/test/java/com/iota/iri/IXITest.java

```diff
@@ -27,6 +32,11 @@ public static void setUp() throws Exception {
         ixiDir.create();
         ixi = new IXI();
         ixi.init(ixiDir.getRoot().getAbsolutePath());
+
+        Field ixiApiField = ixi.getClass().getDeclaredField("ixiAPI");
+        ixiApiField.setAccessible(true);
+        Map<String, Map<String, CallableRequest<AbstractResponse>>> ixiAPI = (Map<String, Map<String, CallableRequest<AbstractResponse>>>) ixiApiField.get(ixi);
```

Comment:
Can you just add a new line after `=` since the line is too long.
I will need to make adjustments so that our static analysis tools will catch those things.

After that change I will merge and squash since the fix and the test are :100: